Cargo.toml

```diff
@@ -149,7 +149,8 @@ datafusion-expr = { git = "https://github.com/spiceai/datafusion.git", rev = "2b
 object_store = { git = "https://github.com/spiceai/arrow-rs.git", rev = "1f37d9eda6da17b7644115f82d209c00c9566733" }
 
 datafusion-federation = { git = "https://github.com/spiceai/datafusion-federation.git", rev = "5af0df83c2cd1d3f82f293b066b401a4dfd4064b" }
-datafusion-table-providers = { git = "https://github.com/datafusion-contrib/datafusion-table-providers.git", rev = "a41d5b2d7b4c4c7abf9bbb777e53b0b9ce8c9f52" }
+# TODO: Change to final commit once https://github.com/datafusion-contrib/datafusion-table-providers/pull/191 is merged
+datafusion-table-providers = { git = "https://github.com/datafusion-contrib/datafusion-table-providers.git", rev = "54f82c8891d90a7da86148d7e96702bd3c9e6c8d" }
```

Comment:
Change once https://github.com/datafusion-contrib/datafusion-table-providers/pull/191 is merged